de

```diff
@@ -7,7 +7,7 @@ GREEN="\e[32m"
 EULER="\e^(iπ)+1=0"
 RESET="\e[0m"
 
-function task { printf "${CYAN}$1... ${RESET}"; }
+function task { printf "${CYAN}$1... ${RESET}\n"; }
```

Comment:
This lacks a newline because there are many places where we print out `Performing some task... done.` I think it looks nicer that way. Do you want to switch to just putting the status on a newline? I'm okay with that if you think it's best.

Reply:
Do you need to? Wouldn't the user already provide the newline?

Reply:
Replaced the call of task with echo to fix that.